samples/csharp_dotnetcore/85.bot-authentication-sni/README.md

```diff
@@ -2,7 +2,13 @@
 
 Bot Framework v4 bot authentication using Subject Name/Issuer
 
-This bot has been created using [Bot Framework](https://dev.botframework.com/), is shows how to use the bot authentication capabilities of Azure Bot Service. In this sample, we use a local or KeyVault certificate and the MSAL Subject Name/Issuer configuration to create the Bot Framework Authentication.
+This bot has been created using Bot Framework, it shows how to use the bot authentication capabilities of Azure Bot Service. In this sample, we use a local or KeyVault certificate and the MSAL Subject Name/Issuer configuration to create the Bot Framework Authentication.
+>NOTE: Microsoft's first-party resources are required to test this sample.
```

Comment:
I will add this NOTE like this. More info in https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts
> [!IMPORTANT]
> Microsoft's first-party resources are required to test this sample.

Reply:
done